packages/SwingSet/src/controller.js

```diff
@@ -397,22 +397,22 @@ export async function buildVatController(
         validateBundleDescriptor(desc, groupName, name);
         if (!desc.bundleName) {
           names.push(name);
-          if (desc.sourcePath) {
-            const sourcePath = desc.sourcePath;
-            if (!(sourcePath[0] === '.' || path.isAbsolute(sourcePath))) {
+          if (desc.sourceSpec) {
+            const sourceSpec = desc.sourceSpec;
+            if (!(sourceSpec[0] === '.' || path.isAbsolute(sourceSpec))) {
```

Comment:
This relative test would pass `.dotfile/boo`, which isn't a relative specifier, it's bare.

Reply:
Good catch.  That bit was cribbed from the code that existing prior to my overhaul of the config subsystem and I sort of blindly copied it without considering it too deeply.

Reply:
Upon further consideration, it's not clear why we should even care in this context.  I will ping Brian.